src/bin/dual-iir.rs

```diff
@@ -64,6 +61,13 @@ const SCALE: f32 = i16::MAX as _;
 // The number of cascaded IIR biquads per channel. Select 1 or 2!
 const IIR_CASCADE_LENGTH: usize = 1;
 
+// The number of samples in each batch process
+const SAMPLE_BUFFER_SIZE: usize = 8;
```

Comment:
Can we seize the opportunity to rename this to a more accurate `BATCH_SIZE`? (It not isn't the size of `SampleBuffer` anymore and it is always the batch size both for ADCs and DACs).

Nice touch of showing the distinction of using a log2 batch size for lockin and that the ticks doesn't need to be log2 unless the app demands it!